projects/assets/src/translations/en/payment.ts

```diff
@@ -49,7 +49,7 @@ export const payment = {
     paymentType_CARD: 'Credit Card',
     paymentType_ACCOUNT: 'Account',
   },
-  messages: {
-    setAsDefaultSuccessfully: 'New payment was successfully set as default',
```

Comment:
Wrong merge.

Reply:
it's the correct one :d, it changed here https://github.com/SAP/spartacus/pull/14776

Reply:
Nope. I fixed the misspelled `Successfully` on our epic at some point.